devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -25,10 +25,12 @@ metadata {
 		capability "Refresh"
 		capability "Health Check"
 		capability "Sensor"
+		
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,0500", outClusters: "0003", manufacturer: "eWeLink", model: "MS01", deviceJoinName: "eWeLink Motion Sensor" //eWeLink Motion Sensor
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001", manufacturer: "ORVIBO", model: "895a2d80097f4ae2b2d40500d5e03dcc", deviceJoinName: "Orvibo Motion Sensor" //Orvibo Motion Sensor
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001,FFFF", manufacturer: "Megaman", model: "PS601/z1", deviceJoinName: "INGENIUM Motion Sensor" //INGENIUM ZB PIR Sensor
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000, 0003, 0500, 0001", outClusters: "0019", manufacturer: "HEIMAN", model: "PIRSensor-N", deviceJoinName: "HEIMAN Motion Sensor" //HEIMAN Motion Sensor
+		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000, 0001, 0500", outClusters: "0019", manufacturer: "Third Reality, Inc", model: "3RMS16BZ", deviceJoinName: "Motion Sensor" //ThirdReality Motion Sensor
```

Comment:
@Jackie-Kilby Usually we put the brand name in the `deviceJoinName` like "Third Reality Motion Sensor". Do you want to do that or leave it more generic?

Reply:
Thank you. I will use "ThindReality Motion Sensor". However, may I state my opinion? 
I think a user will pair only one or two sensors, or smart devices once a time. As a normal user, he do care the brand of the device when purchasing. However, he do not care what the brand is when he is pairing the device. So I think all deviceJoinName should be generic, only the manufacturer info will tell the difference backstage.

Reply:
@Jackie-Kilby thanks for the feedback. I understand what you mean but this was a thoughtful decision and it's done this way for most devices so if nothing else it's helpful for consistency.